src/cpp/r/session/graphics/RGraphicsDevDesc.cpp

```diff
@@ -264,11 +264,47 @@ pDevDesc allocate(const RSDevDesc& devDesc)
 
        return (pDevDesc) pDD;
    }
+
+   case 16:
+   {
+      DevDescVersion16* pDD =
+           allocAndInitCommonMembers<DevDescVersion16>(devDesc);
+
+      pDD->path = devDesc.path;
+      pDD->raster = devDesc.raster;
+      pDD->cap = devDesc.cap;
+      pDD->eventEnv = devDesc.eventEnv;
+      pDD->eventHelper = devDesc.eventHelper;
+
+      pDD->holdflush = devDesc.holdflush;
+      pDD->haveTransparency = devDesc.haveTransparency;
+      pDD->haveTransparentBg = devDesc.haveTransparentBg;
+      pDD->haveRaster = devDesc.haveRaster;
+      pDD->haveCapture = devDesc.haveCapture;
+      pDD->haveLocator = devDesc.haveLocator;
+
+      pDD->setPattern = devDesc.setPattern;
+      pDD->releasePattern = devDesc.releasePattern;
+      pDD->setClipPath = devDesc.setClipPath;
+      pDD->releaseClipPath = devDesc.releaseClipPath;
+      pDD->setMask = devDesc.setMask;
+      pDD->releaseMask = devDesc.releaseMask;
+
+      pDD->defineGroup = devDesc.defineGroup;
+      pDD->useGroup = devDesc.useGroup;
+      pDD->releaseGroup = devDesc.releaseGroup;
+      pDD->stroke = devDesc.stroke;
+      pDD->fill = devDesc.fill;
+      pDD->fillStroke = devDesc.fillStroke;
+      pDD->capabilities = devDesc.capabilities;
+
```

Comment:
```suggestion

pDD->glyph = devDesc.glyph;
```

Reply:
I woke up and realized I think I forgot something!